lib/Froxlor/Settings/Store.php

```diff
@@ -367,4 +367,62 @@ public static function storeSettingWebserverFcgidFpmUser($fieldname, $fielddata,
 
 		return $returnvalue;
 	}
+
+	public static function storeSettingImage($fieldname, $fielddata)
+    {
+        if (isset($fielddata['settinggroup'], $fielddata['varname']) && is_array($fielddata) && $fielddata['settinggroup'] !== '' && $fielddata['varname'] !== '') {
+            $save_to = null;
+            $path = \Froxlor\Froxlor::getInstallDir().'/img/';
+
+            // New file?
+            if ($_FILES[$fieldname]['tmp_name']) {
```

Comment:
Added a proper mime-check now, however, it was already pretty hard to do so since the upload file selector was limited to jpg/png already and it requires an admin-login in the panel in the first place.

The naming of those files is already static and set in the field settings, the user cannot change that.

Reply:
I know...I just remembered the guy reporting that "possible security issue" when installing froxlor and having the userdata file in /tmp etc.etc.etc. totally overkill but ...still a potential way so, better safe than sorry

Reply:
Yeah, added the mime-type check in the last commit.